SPECS/python-mock/python-mock.spec

```diff
@@ -0,0 +1,200 @@
+# keeping python3 subpackage as stdlib mock lives in a different namespace
+# Some people may have not fixed their imports
+
+%global mod_name mock
+
+Summary:        A Python Mocking and Patching Library for Testing
+Name:           python-mock
+Version:        3.0.5
+Release:        9%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            http://www.voidspace.org.uk/python/mock/
+#Source0:        https://github.com/testing-cabal/mock/archive/refs/tags/%{version}.tar.gz
+Source0:        mock-%{version}.tar.gz
```

Comment:
>mock [](http://example.com/codeflow?start=16&length=4)

Can we make it `%{name}-%{version}` instead?

Reply:
What I had in mind is to call the tarball `python-mock-3.0.5.tar.gz`, since just `mock-...` is seem more colision-prone for our source repo.:) Thys the ask for `%{name}` instead of `%{mod_name}`.